src/Microsoft.Azure.WebJobs.Host/Blobs/ParameterizedBlobPath.cs

```diff
@@ -21,16 +31,33 @@ public ParameterizedBlobPath(BindingTemplate containerNameTemplate, BindingTempl
 
             _containerNameTemplate = containerNameTemplate;
             _blobNameTemplate = blobNameTemplate;
+            _urlBindingTemplate = null; // does not need to initialize, since we don't have property
         }
 
         public string ContainerNamePattern
         {
-            get { return _containerNameTemplate.Pattern; }
+            get
+            {
+                if (_urlBindingTemplate != null)
+                {
+                    // return template string for container
+                    return _urlBindingTemplate.Pattern + ".getContainer()";
```

Comment:
string representation of a `urlTemplate`is something like `{data.url}`, so a string representation for `containerTemplate` can be `{data.url}.getContainer()`

Reply:
I don't understand.  That's not a valid value for this method to return. 

---
In reply to: [135114358](https://github.com/Azure/azure-webjobs-sdk/pull/1315#discussion_r135114358) [](ancestors = 135114358)

Reply:
I see  it looks like these results are only used for logging, so we could get away with returning anything. Can you update the comment to make that clear? 

---
In reply to: [135612801](https://github.com/Azure/azure-webjobs-sdk/pull/1315#discussion_r135612801) [](ancestors = 135612801,135114358)